deploy/examples/create-external-cluster-resources.py

```diff
@@ -471,7 +474,7 @@ def gen_arg_parser(cls, args_to_parse=None):
             "--upgrade",
             action="store_true",
             default=False,
-            help="Upgrades the cephCSIKeyrings(For example: client.csi-cephfs-provisioner) with new permissions needed for the new cluster version and older permission will still be applied."
+            help="Upgrades the cephCSIKeyrings(For example: client.csi-cephfs-provisioner) and client.healthchecker ceph users with new permissions needed for the new cluster version and older permission will still be applied."
```

Comment:
the same thing needs to be updated in the external-cluster crd document.

Reply:
updated